shardingsphere-features/shardingsphere-shadow/shardingsphere-shadow-api/src/main/java/org/apache/shardingsphere/shadow/api/config/table/ShadowTableConfiguration.java

```diff
@@ -46,4 +46,16 @@ public ShadowTableConfiguration(final Collection<String> shadowAlgorithmNames) {
         this.dataSourceNames = new LinkedList<>();
         this.shadowAlgorithmNames = shadowAlgorithmNames;
     }
+    
+    /**
+     * Aggregate the data of two shadow table configuration.
+     *
+     * @param config new shadow table configuration
+     * @return shadow table configuration
+     */
+    public ShadowTableConfiguration aggregateData(final ShadowTableConfiguration config) {
```

Comment:
Data is not a good name.

Reply:
I also agree, it has been modified to `mergeConfiguration()` and moved to `ShadowRuleStatementSupporter`, What do you think of this？